net/haproxy/src/opnsense/mvc/app/models/OPNsense/HAProxy/HAProxy.xml

```diff
@@ -620,6 +620,13 @@
                         <uri>URI Hash (only HTTP mode)</uri>
                     </OptionValues>
                 </algorithm>
+                <proxyProtocol type="OptionField">
+                    <Required>N</Required>
+                    <OptionValues>
+                        <v1>PROXY protocol version 1</v1>
+                        <v2>PROXY protocol version 2</v2>
```

Comment:
maybe just "Version 1" because the label is already "Proxy Protocol"

Reply:
Fixed.